packages/Webkul/Installer/src/Console/Commands/Installer.php

```diff
@@ -235,16 +246,25 @@ protected static function getEnvAtRuntime(string $key): string|bool
     protected function createAdminCredential()
     {
         // Here! Asking for an admin name, email, and password
-        $adminName = $this->ask('Please enter the <bg=green>Name of Admin User</> or press enter to continue', 'Example');
-        $adminEmail = $this->ask('Please enter the <bg=green>Email of Admin User</> or press enter to continue', 'admin@example.com');
-
-        if (! filter_var($adminEmail, FILTER_VALIDATE_EMAIL)) {
+        $adminName = text(
+            label: 'Enter the name of the admin user',
+            default: 'Example',
+            required: true
+        );
+
+        $adminEmail = text(
+            label: 'Enter the email address of the admin user',
+            default: 'admin@example.com',
+            required: true
+        );
+
+        if (!filter_var($adminEmail, FILTER_VALIDATE_EMAIL)) {
             $this->error('The email address you entered is not valid please try again.');
 
             return $this->createAdminCredential();
         }
 
-        $adminPassword = $this->ask('Please enter the <bg=green>Password</> for admin login:', 'admin123');
+        $adminPassword = Str::password();
```

Comment:
Password will always be 'admin123'.

Reply:
I changed it back as requested to text and with a default value admin123, however i don't think that this is a secure way to setup the application.